composer.json

```diff
@@ -5,26 +5,31 @@
     "description": "Bundle that provides controllers and services for using Sulu as headless content management system",
     "require": {
         "php": "^7.2",
-        "sulu/sulu": "^2.0.1"
+        "sulu/sulu": "^2.0.1",
+        "symfony/config": "^4.3 || ^5.0",
+        "symfony/dependency-injection": "^4.3 || ^5.0",
+        "symfony/framework-bundle": "^4.3 || ^5.0",
+        "symfony/http-foundation": "^4.3 || ^5.0",
+        "symfony/http-kernel": "^4.3 || ^5.0"
     },
     "require-dev": {
-        "coduo/php-matcher": "^3.2",
-        "handcraftedinthealps/code-coverage-checker": "^0.2.0",
+        "coduo/php-matcher": "^3.2 || ^4.0",
+        "handcraftedinthealps/code-coverage-checker": "^0.2.1",
         "handcraftedinthealps/zendsearch": "^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.3.4",
-        "jangregor/phpstan-prophecy": "^0.4.1",
-        "php-ffmpeg/php-ffmpeg": "^0.13 || ^0.14",
-        "phpstan/phpstan": "^0.11.12",
-        "phpstan/phpstan-doctrine": "^0.11.5",
-        "phpstan/phpstan-phpunit": "^0.11.2",
-        "phpstan/phpstan-symfony": "^0.11.6",
+        "jangregor/phpstan-prophecy": "^0.8",
+        "phpstan/phpstan": "^0.12",
+        "phpstan/phpstan-doctrine": "^0.12",
+        "phpstan/phpstan-phpunit": "^0.12",
+        "phpstan/phpstan-symfony": "^0.12",
         "phpunit/phpunit": "^8.0",
         "symfony/browser-kit": "^4.3 || ^5.0",
+        "symfony/console": "^4.3 || ^5.0",
+        "symfony/debug": "^4.3",
         "symfony/dotenv": "^4.3 || ^5.0",
         "symfony/monolog-bundle": "^3.1",
-        "thecodingmachine/phpstan-strict-rules": "^0.11.2"
+        "thecodingmachine/phpstan-strict-rules": "^0.12"
     },
-    "minimum-stability": "dev",
```

Comment:
I would not remove this, its always good to get reminded as soon as possible about incompatibility in new versions.  @nnatter what do you think?

Reply:
fixed

Reply:
dont have a strong opinion - would try to be consistent with other bundles 🙂  what are we using there?